src/homeworks/homework-3.ts

```diff
@@ -1,93 +1,175 @@
-class School {
+class NewSchool {
```

Comment:
I had an error `TS2300: Duplicate identifier School. homework-2.ts(1, 7):  School  was also declared here.` so I renamed all the classes. is it ok?

Reply:
Yes, it's totally fine.